bitcoin/src/merkle_tree/block.rs

```diff
@@ -557,11 +561,7 @@ mod tests {
 
     #[cfg(feature = "rand-std")]
     fn pmt_test(tx_count: usize) {
-        use core::cmp::min;
-
-        use crate::merkle_tree;
-
-        let mut rng = thread_rng();
+        let mut rng = prelude::thread_rng();
```

Comment:
So this is one case where typically we break the use-one-level-of-path rule and just call the function as it was.

Reply:
Changed back to just call the function